### ISSUE TITLE
Symfony3 - Could not load type "text"

### ISSUE THREAD

**Issue by cklm** [PUGX/PUGXAutoCompleterBundle#30]
Hi, I'm using your bundle with Symfony3. 

This is my code:
```
public function buildForm(FormBuilderInterface $builder, array $options) {
        $builder->add('book',AutocompleteType::class,array(
            'class'=>'AppBundle:Book'
        ));
}
```

When I want to use it, an error500 happens, saying Could not load type "text" (in vendor\symfony\symfony\src\Symfony\Component\Form\FormRegistry.php at line 87). 
I guess, it has something to do with getParent() which returns 'text'. Is there a fix for that?

**Comment by garak** [comment]
Fixed in 40cbc7d

**Comment by cklm** [comment]
That was quick - thank you very much!
